### DISCUSSION TITLE
Count wont work

### DISCUSSION THREAD

**Discussion by Kola-Ademola** [sidorares/node-mysql2#1540] [Q&A]
Am trying to get another count function for where "qty_count" > 2 buy I keep getting a syntax error ![image](https://user-images.githubusercontent.com/60304620/159448363-d85c477c-6f37-4456-981b-a420f324170f.jpeg)

**Comment by sidorares** [comment]
while you might get some help here @Kholaaa I believe it's a very generic sql question not directly related no node-mysql2 library. https://stackoverflow.com/questions/tagged/sql or https://stackoverflow.com/questions/tagged/mysql might be a better forum to ask answer to your question.

Also posting exact error text might help ( I can see only part of it in the screenshot )

**Reply by Kola-Ademola** [reply]
Ok thanks
